apps/server/src/modules/board/uc/column.uc.ts

```diff
@@ -42,6 +42,7 @@ export class ColumnUc extends BaseUc {
 		this.logger.debug({ action: 'createCard', userId, columnId });
 
 		const column = await this.columnService.findById(columnId);
+		// TODO students should not be allowed to create cards
 		await this.checkPermission(userId, column, Action.read);
```

Comment:
This should be changed to `Action.write`

Reply:
this is a bug which should be fixed. But is not related to our ticket